.github/workflows/ci.yaml

```diff
@@ -0,0 +1,101 @@
+name: CI
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+  schedule:
+    - cron: "0 13 * * 1"
+
+codecov:
+  require_ci_to_pass: no
+  max_report_age: off
+
+comment: false
+
+coverage:
+  precision: 2
+  round: down
+  status:
+    project:
+      default:
+        target: 95
+        informational: true
+    patch: off
+    changes: off
+
```

Comment:
This is meant to be in `codecov.yml` file at the root directory of the repo

Reply:
Ahhhh, sorry that was stupid of me.